clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/KubernetesV2Credentials.java

```diff
@@ -149,10 +150,13 @@ public KubernetesV2Credentials(
                           return new KubernetesKindProperties(k, true, false, true);
                         }))
             .map(KubernetesKindProperties::getKubernetesKind)
-            .collect(Collectors.toList());
+            .collect(ImmutableSet.toImmutableSet());
```

Comment:
Yeah, I tend to avoid static imports as I find myself always scrolling up to figure out where something came from, though I suppose this is one of the more common ones.